python/semantic_kernel/planning/stepwise_planner/Skills/StepwiseStep/skprompt.txt

```diff
@@ -24,6 +24,18 @@ Here is an example of a valid $JSON_BLOB:
   "action": "functionName",
   "action_variables": {"parameterName": "some value", ...}
 }
+
+Here is another example of a valid $JSON_BLOB:
```

Comment:
Good to know! Out of curiosity, is there way to unit test prompts?

Reply:
I heard people develop the idea of testing LLM prompts with another LLM.

In this case, how do you test your change? By running the notebooks?

Reply:
We are not unit testing them at the moment AFAIK, but yes, we can write unit tests around prompts. If we do, we should consider testing them against all the LLMs SK supports to ensure consistency in the responses.

Reply:
By running a script provided by the user in #3237 and comparing the results from `SequentialPlanner` which worked out of the box vs `StepwisePlanner` which wasn't working.

Reply:
This would have to be a integration test since the unit tests do not interact with outside services, but having testing for plans, both creation and execution would be quite helpful.